curriculum/challenges/english/01-responsive-web-design/css-piano/part-010.md

```diff
@@ -0,0 +1,81 @@
+---
+id: 612e89d254fe5d3df7d6693d
+title: Part 10
+challengeType: 0
+dashedName: part-10
+---
+
+# --description--
+
+Now that you have reset the `html` box model, you need to pass that on to the elements within as well. To do this, you can set the `box-sizing` property to `inherit`, which will tell the targeted elements to use the same value as the parent element.
+
+You will also need to target the pseudo-elements, which are special keywords that follow a selector. Pseudo-elements allow you to style specific parts of the targeted element.
```

Comment:
This is the first time Campers are being introduced to pseudo-elements. So, I think it might be best to mention what `::before` is, and what `::after` is. That is, the current description adds nothing to my understanding - I am a long-time webdeveloper, and I have little idea of what these selectors are.

What are pseudo-elements? And, how does _targetting_ them do anything to my elements?